tests/GenericSparqlTest.php

```diff
@@ -738,7 +781,7 @@ public function testQueryParentList()
     );
     $props = array (
       'uri' => 'http://www.skosmos.skos/test/ta1',
-      'top' => 'http://www.skosmos.skos/test/conceptscheme',
```

Comment:
Please don't drop the definition of `top`, it is still needed for backwards compatibility.